js/index.js

```diff
@@ -1354,8 +1357,10 @@ function DataManager() {
 			});
 		} else {
 			oTable.fnClearTable();
-			oTable.fnAddData(dataList);
-			oTable.fnAdjustColumnSizing();
+			if (dataList.length > 0){
+				oTable.fnAddData(dataList);
+				oTable.fnAdjustColumnSizing();
+			}
```

Comment:
I pasted you the url in the PR comment (See example). Here it is again anyway: [empty table example](https://sandbox.psi-mis.org/api/apps/datadictionary/index.html?Tab=DataSet&Value=dZmdh5abx4b). I will ask for a DB dump but I would postpone this task for the moment unless you find a quick fix. Do not want to waste too much time on this. If you can not find a quick fix we can maybe create a card that sums up this issue and we can come back to it in the next sprint. Let me know...